environment.yml

```diff
@@ -2,12 +2,17 @@ name: pyleco
 channels:
   - conda-forge
 dependencies:
-  - pyzmq=25.0.0
+  - pyzmq=25.1.0
+  - pip  # don't pin, to gain newest conda compatibility fixes
+  - pip:
+    - jsonrpc2-pyclient==2.2.12
+    - openrpc==6.3.17
+    - uuid7==0.1.0
 # Development dependencies below
   - pytest=7.2.0
+  - pytest-cov=4.1.0
+  - pytest-runner=6.0.0
```

Comment:
add `ruff`?

Reply:
Not needed to install ruff into our python installation, as we use a "ruff action".

Reply:
Ah nice!